pkg/tfgen/generate_nodejs_utilities.go

```diff
@@ -0,0 +1,37 @@
+package tfgen
+
+const tsUtilitiesFile = `
+export function getEnv(...vars: string[]): string | undefined {
+    for (const v of vars) {
+        const value = process.env[v];
+        if (value) {
+            return value;
+        }
+    }
+    return undefined;
+}
+
+export function getEnvBoolean(...vars: string[]): boolean | undefined {
+    const s = getEnv(...vars);
+    if (s !== undefined) {
+        if (["1", "t", "T", "true", "TRUE", "True"].find(v => v === s) !== undefined) {
```

Comment:
I guess this list comes from TF (or from Go library behaviour somewhere?)?  Can we cross reference with the place this is defined?

Reply:
This comes from Go library behavior. I will add a reference to the relevant function.